_episodes/00-before-we-start.md

```diff
@@ -147,7 +147,7 @@ default.
 
 Since we want our code and workflow to be reproducible, it is better to type the commands in
 the script editor, and save them as a script. This way, there is a complete record of what we did,
-and anyone (including our future selves!) can easily reproduce the results on their computer.
+and anyone (including our future selves!) has an easier time reproducing the results on their computer.
```

Comment:
Perhaps: can easily reproduce -> can  reproduce

Reply:
I thought about that, but none of the advice makes it 100% reproducible, and since "easier" is used more often, it seems more fitting to me.

Reply:
oh, the curse of knowledge... you can leave it as is but here are my 2 cents:

you're certainly correct. However, sometimes we have to sacrifice truth for clarity. In this case, we're presenting learners a new (truthful) idea that it's better to use scripts. It is, therefore, better to use simple and convincing language. The message that learners will remember will be true. Unless they're ~cursed~ blessed with perfect memory, they won't come back saying "...but you said my results will be reproducible..."